libraries/AP_RCProtocol/AP_RCProtocol_FPort.cpp

```diff
@@ -124,8 +128,8 @@ void AP_RCProtocol_FPort::decode_control(const FPort_Frame &frame)
 
     bool failsafe = ((frame.control.flags & (1 << FLAGS_FAILSAFE_BIT)) != 0);
 
-    // we scale rssi by 2x to make it match the value displayed in OpenTX
-    const uint8_t scaled_rssi = MIN(frame.control.rssi*2, 255);
+    // fport rssi 0-50, ardupilot rssi 0-255, scale factor 255/50=5.1
```

Comment:
Actually, we're trying to move to `0.0-1.0` for RSSI in ArduPilot itself.  Some interfaces in might be 0-255, 'though


https://github.com/ardupilot/ardupilot/blob//libraries/GCS_MAVLink/GCS_Common.cpp#L603

Reply:
yes but we need to address the fport rssi issue, right now it's broken